site/app/Resources/views/template/email_template.html.twig

```diff
@@ -0,0 +1,10 @@
+<html>
+<body>
+<div class="container content">
+    <h4 class="salutation">Hi {{ user }},</h4>
+    <p class="body">You have a new notification from Librecores website. {{ content }}</p>
```

Comment:
Put content in a separate DIV. 

Use this:  https://github.com/rodriguezcommaj/salted as a base for all emails. We already use it in password reset emails